### PR TITLE
New create modal

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- Add new create modal and use it for Deployments create.
- ALSO fix title changing when the Preferences view is opened.

**Related issue(s)**

### CODE REVIEW

File: core-ui/src/shared/components/CreateModal/Presets.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  const { t } = useTranslation();
+  const options = presets.map(({ name }) => ({
+    key: name,
+    text: name,
+  }));
+
+  return (
+    <Dropdown
+      placeholder={t('deployments.create-modal.presets.choose')}
+      compact
```
Comment (akucharska) [inline core-ui/src/shared/components/CreateModal/Presets.js:15]: Should we remove compact from this Dropdown or make all other fields compact?

File: core-ui/src/shared/components/CreateDeploymentForm/SimpleForm.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          i18n={i18n}
+        />
+        <LabelsInput
+          labels={deployment.labels}
+          onChange={labels => setDeployment({ ...deployment, labels })}
+          i18n={i18n}
+        />
+        <FormItem>
+          <FormLabel htmlFor="docker-image" required>
+            {t('deployments.create-modal.simple.docker-image')}
```
Comment (akucharska) [inline core-ui/src/shared/components/CreateDeploymentForm/SimpleForm.js:34]: Should we have labels next to the form fields?

**Review by akucharska — COMMENTED** [review]
Great job :) I only have two questions/suggestions

**Review by akucharska — APPROVED** [review]

File: core/src/i18n/en.yaml [hunk 3]
```diff
@@ -164,6 +170,29 @@ daemon-sets:
...
 deployments:
+  create-modal:
+    advanced:
+      cpu-limits: CPU limits
+      cpu-requests: CPU requests
+      expose-service: Expose separate Service
+      memory-limits: Memory limits
+      memory-requests: Memory requests
+      port: Port
+      port-placeholder: Enter port at which expose the Service
```
Comment (NHingerl) [inline core/src/i18n/en.yaml:181]: Does the Service expose the port, or does the port expose the Service?
Depending on that, I'd suggest either

`port-placeholder: Enter the port that exposes the Service`

or

`port-placeholder: Enter the port exposed by the Service`

File: core/src/i18n/en.yaml [hunk 4]
```diff
@@ -164,6 +170,29 @@ daemon-sets:
...
+      runtime-profile: Runtime profile
+      target-port: Target port
+      target-port-placeholder: Enter target port of the container
```
Comment (NHingerl) [inline core/src/i18n/en.yaml:184]:
```suggestion
      target-port-placeholder: Enter the target port of the container
```

File: core/src/i18n/en.yaml [hunk 5]
```diff
@@ -164,6 +170,29 @@ daemon-sets:
...
+    messages:
+      deployment-ok-service-bad: Deployment created, failed to create the Service
+      failure: Failed to create the Deployment
+      success: Deployment created
+    presets:
+      choose: Select preset
+      default: Default
+    simple:
+      docker-image: Docker image
+      docker-image-placeholder: Enter Docker image tag, e.g. bitnami/nginx
```
Comment (NHingerl) [inline core/src/i18n/en.yaml:194]:
```suggestion
      docker-image-placeholder: Enter Docker image tag, for example, bitnami/nginx
```

File: core/src/i18n/en.yaml [hunk 6]
```diff
@@ -164,6 +170,29 @@ daemon-sets:
     title: Tolerations
     toleration-seconds: Seconds
 deployments:
+  create-modal:
+    advanced:
```
Comment (NHingerl) [inline core/src/i18n/en.yaml:174]: If the following strings are headers, consider capitalization - see [English: General Rule for Capitalization (Sentence Style and Title Case) on the User Interface](https://help.sap.com/viewer/DRAFT/e33c591ae4494a659a3f5f983c9d1161/PROD/en-US/79ce282052d14cc38e6a203a1fc277c4.html).

**Review by NHingerl — DISMISSED** [review]

**Review by NHingerl — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Pull request merged**